go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.3
 
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
-	github.com/cloudflare/cloudflare-go v0.114.0
+	github.com/cloudflare/cloudflare-go v0.115.0
```

Comment:
@sebassimoes please remove the version bumps per the automation. we handle these outside of feature PRs to prevent merge conflict hell.